### PR TITLE
Add list querying

### PR DESCRIPTION
**Pull request by marioreggiori**
In addition to #43

### CODE REVIEW

File: lib/src/objectdb_base.dart [hunk 1]
```diff
@@ -505,7 +535,8 @@ class ObjectDB extends CRUDController {
           query[i] == null) {
         prepared[key] = query[i];
       } else {
-        throw ObjectDBException("query contains invalid data type '${query[i]?.runtimeType}'");
+        throw ObjectDBException(
+            "query contains invalid data type '${query[i]?.runtimeType}'");
```
Comment (flexwie) [inline lib/src/objectdb_base.dart:539]: why lowercase

File: test/objectdb_test.dart [hunk 2]
```diff
@@ -134,7 +134,9 @@ void main() async {
     print('remove count ' + removeCount.toString());
   });
 
-  test('operators', () async {});
+  test('operators', () async {
+    // todo
```
Comment (flexwie) [inline test/objectdb_test.dart:138]: uhhhhhhmmmmm

**Review by flexwie — CHANGES_REQUESTED** [review]
Error message should be made consistent

File: test/objectdb_test.dart (hunk 2 above)
Comment (marioreggiori) [reply test/objectdb_test.dart:138]: still todo

File: lib/src/objectdb_base.dart (hunk 1 above)
Comment (marioreggiori) [reply lib/src/objectdb_base.dart]: Fixed with 8e95ba1

**Review by flexwie — APPROVED** [review]

**Pull request merged**